app/models/user.rb

```diff
@@ -4,9 +4,8 @@ class User < ApplicationRecord
   has_one_attached :profile_picture
   validates :full_name, :gender, :country, :city, :username, presence: true
   validates :username, uniqueness: true
+  validates :password, :password_confirmation, presence: true
```

Comment:
Why this change is here in this Pull request and it is also present in [Tron-14](https://github.com/Hassan0112/Trons/pull/17/files) ???? we don't want any unecessary changes other than whats required in the ticket